packages/wonder-blocks-dropdown/components/select-item.js

```diff
@@ -47,10 +46,10 @@ type SelectProps = {
 
     /**
      * Callback for when this item is pressed to change its selection state.
-     * Passes value of the item and the new selection state of the item. Should
+     * Passes value of the item, its label, and its new selection state. Should
      * be handled by an implementation of Menu.
      */
-    onToggle: (value: string, selectionState: boolean) => void,
+    onToggle: (value: string, label: string, selectionState: boolean) => void,
```

Comment:
The `label` doesn't seem all that useful since whatever component is passing down data can always look up the `label` based on the `value`.